bench/irmin-pack/trace_replay_intf.ml

```diff
@@ -101,8 +101,14 @@ module type Store = sig
   val create_repo :
     root:string -> store_config -> (Repo.t * on_commit * on_end) Lwt.t
 
-  val gc : repo -> commit_key -> unit Lwt.t
-  val finalise_gc : ?wait:bool -> repo -> bool Lwt.t
+  val gc_wait : repo -> unit Lwt.t
+  val gc_is_finished : repo -> bool
```

Comment:
It seems that `gc_is_finished` is not used. Does this mean that this can be reviewed?

Reply:
Good catch. I think I used this at one point along the way. Will remove.